pkg/connector/project.go

```diff
@@ -67,15 +64,9 @@ func (p *projectBuilder) List(ctx context.Context, parentResourceID *v2.Resource
 		PerPage: ITEMSPERPAGE,
 		Page:    pageToken,
 	})
+	err = getError(err)
```

Comment:
Doesn't this code want `checkStatusUnauthorizedError()`?

Reply:
Fixed